# Don't scan binary & media files
# Ugly, not upstreamable hack

diff -up scancode-toolkit-2.0.1/src/scancode/cli.py.ignore-binary scancode-toolkit-2.0.1/src/scancode/cli.py
--- scancode-toolkit-2.0.1/src/scancode/cli.py.ignore-binary	2017-07-03 18:02:17.000000000 +0200
+++ scancode-toolkit-2.0.1/src/scancode/cli.py	2017-07-27 13:26:07.076096747 +0200
@@ -621,6 +621,9 @@ def _scanit(paths, scanners, scans_cache
     infos['path'] = rel_path
     infos.update(scan_infos(abs_path, diag=diag))
 
+    if any((infos['is_binary'], infos['is_media'], infos['size'] > 1000000)):
+        return True, rel_path
+
     success = True
     scans_cache = scans_cache_class()
     is_cached = scans_cache.put_info(rel_path, infos)
